dev_docs/tutorials/development_windows.mdx

```diff
@@ -0,0 +1,147 @@
+summary: Learn how to setup a Windows development environment using WSL
+date: 2022-06-16
+tags: ['kibana', 'onboarding', 'setup', 'windows', 'development', 'wsl']
+---
+
+
+# Overview
+
+Development on Windows is recommended through WSL2.  WSL lets users run a Linux environment on Windows, providing a supported development environment for Kibana.
+
+<DocCallOut title="Prerequisites">
+  
+  ### Basic Experience
+  For a basic WSL usage you must be running **Windows 10 version 2004 and higher (Build 19041 and higher) or Windows 11**. Note that in a basic setup you won't be able to use 
+  the browser GUI to visually run and/or debug functional tests. It's highly advisable to make sure you install the prerequisites for a default usage listed below.
+  
+  **Note:** For a normal development cycle we really advise to use the default experience setup within Windows 11. However if you have an explicit need for using a browser GUI 
+  while using Windows 10, please follow each step of the guide below applicable for the Basic Experience setup and then check our <DocLink id="kibDevTutorialSetupWindowsDevWSL" section="short-term-workaround-for-windows-10-gui-support" text="short term workaround for Windows 10 GUI support" />.
+  
+  ### Default Experience
+  For a default experience and a completion of the entire guide which will make you able to run GUI apps you must be running **Windows 11 (Build 22000 and higher)**.
+  In addition to that, installing a **vGPU driver** is also required. Please choose the one compatible with your hardware: 
+
+  - [Intel GPU driver for WSL](https://www.intel.com/content/www/us/en/download/19344/intel-graphics-windows-dch-drivers.html?)
+  - [AMD GPU driver for WSL](https://www.amd.com/en/support/kb/release-notes/rn-rad-win-wsl-support)
+  - [NVIDIA GPU driver for WSL](https://developer.nvidia.com/cuda/wsl)
+  
+  More information for that could be found in the [official guide for GUI apps on WSL](https://docs.microsoft.com/en-us/windows/wsl/tutorials/gui-apps#install-support-for-linux-gui-apps).
+</DocCallOut>
+
+## Install WSL
+
+The latest setup instructions can be found at https://docs.microsoft.com/en-us/windows/wsl/install
+
+1. Open Powershell as an administrator
+
+2. Install WSL with Ubuntu and restart the machine when prompted
+   ```
+   wsl --install -d Ubuntu
+   ```
+
+   In case you already have WSL installed you can just `wsl --update` and then reboot the machine.
+
+3. Once rebooted the installation will proceed for a new WSL install. If that's not the case make sure you're using 
+   WSL2 and Ubuntu by
+   ```
+   wsl --set-default-version 2
+   ```
+
+   and
+
+   ```
+   wsl --set-default Ubuntu 
+   ```  
+
+4. Install support for Google Chrome by running the following inside WSL shell **(IMPORTANT: <DocLink id="kibDevTutorialSetupWindowsDevWSL" section="default-experience" text="Default Experience Only"/>)**
+
+   ```
+   sudo apt update && sudo apt -y upgrade && sudo apt -y autoremove 
```

Comment:
I think we should separate the first `sudo apt update` so that people can input their superuser password and then include the rest of the commands.

Reply:
When I ran this chunk of commands in the terminal the first line executed and asked me for my password, but then the subsequent lines were fed in as the password, leading to several "incorrect password" prompts and then I had to copy & paste the subsequent lines a second time. I think we should anticipate this for other users and separate the commands into two steps

Reply:
Good finding, changed!